content/event/004.open-tech-meetup-3.de.md

```diff
@@ -0,0 +1,33 @@
+---
+published: 2025-03-20
+description: Dresden's Network-Community comes trifft sich zum Dritten Mal.
+title: Drittes DD-IX Open Tech Meetup
+keywords: [ Tech, Update, Event ]
+authors: [ Maurice, Tassilo ]
+image: 004_open_tech_3.jpg
+start_time: 2025-05-06T18:00:00+02:00
+end_time: 2025-05-06T23:00:00+02:00
+location: Andreas-Pfitzmann-Bau (APB) Raum E023
+---
+
+Acht Monate sind vergangen, seit die ersten beiden Peers angefangen haben Pakete auszutauschen, und die Zahl unserer Peers ist stetig gewachsen. Daher möchten wir zunächst unsere neuen Peers begrüßen und einen Ausblick auf unsere geplanten Projekte geben. Außerdem werden wir aktuelle Software-Innovationen und unsere Zusammenarbeit mit ALASCA e.V. vorstellen. Schließlich möchten wir den Abend in entspannter Atmosphäre gemeinsam mit unseren Peers in einem Restaurant ausklingen lassen.
```

Comment:
Ich denke das peers im letzten Satz funktioniert nur im englischen mit der doppeldeutigkeit im deutschen sind es nur unser Kunden und keine Freunde.

Reply:
Hier stimmt die Perspektive nicht. Wir wollen nicht die untenstehende Agenda in Fliesstext formulieren ("zunaechst"). Davon unabhaengig wuerde ich es kuerzer halten:

Die feierliche Eröffnung des DD-IX liegt keine vier Monate zurück. In der Zwischenzeit ist wieder viel passiert. Neue Peers, neue Technik, neue Projekte! Auf unserem 3. Open Tech Meetup wollen wir uns in gewohnter Umgebung mit Euch darüber austauschen.

Reply:
> Ich denke das peers im letzten Satz funktioniert nur im englischen mit der doppeldeutigkeit im deutschen sind es nur unser Kunden und keine Freunde.

der Satz ist schon auch deswegen durcheinander, weil ja die Peers angesprochen werden (Liebe Peers ... gemeinsam mit unseren Peers ...).